sdks/wasm/sdk/NuGet.config

```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+	<packageSources>
+		<add key="local" value="packages" />
```

Comment:
This is giving the following error `The local source 'mono/sdks/wasm/sdk/packages' doesn't exist.`  Is there a way to create this automatically during the restore instead of manually?

Reply:
Weird, here do you get that error? I don't see it even with a clean build.

Reply:
Humm, not getting this anymore.  Weird.